Common/Securities/CryptoFuture/CryptoFutureMarginModel.cs

```diff
@@ -16,6 +16,7 @@
 using System;
 using System.Linq;
 using QuantConnect.Orders;
+using QuantConnect.Securities.Option;
```

Comment:
`using QuantConnect.Securities.Option;` we don't need this guy, same for `Common/Securities/CryptoFuture/CryptoFutureHolding.cs`